pkg/apis/kops/v1alpha2/instancegroup.go

```diff
@@ -176,6 +179,13 @@ const (
 // SpotAllocationStrategies is a collection of supported strategies
 var SpotAllocationStrategies = []string{SpotAllocationStrategyLowestPrices, SpotAllocationStrategyDiversified, SpotAllocationStrategyCapacityOptimized}
 
+// InstanceMetadataOptions holds the fields required to configure instance metadata in an instance.
+// This currently holds fields to require EC2 Instance Metadata in EC2
+type InstanceMetadataOptions struct {
+	HTTPPutResponseHopLimit *int64  `json:"httpPutResponseHopLimit,omitempty"`
+	HTTPTokens              *string `json:"httpTokens,omitempty"`
+}
+
```

Comment:
Should match the other API file.
```suggestion
// InstanceMetadata defines the EC2 instance metadata service options (AWS Only)
type InstanceMetadataOptions struct {
	// HTTPPutResponseHopLimit is the desired HTTP PUT response hop limit for instance metadata requests.
	// The larger the number, the further instance metadata requests can travel. The default value is 1.
	HTTPPutResponseHopLimit *int64 `json:"httpPutResponseHopLimit,omitempty"`
	// HTTPTokens is the state of token usage for the instance metadata requests.
	// If the parameter is not specified in the request, the default state is "optional".
	HTTPTokens *string `json:"httpTokens,omitempty"`
}

```